Jenkinsfile

```diff
@@ -18,7 +18,7 @@ node('docker') {
         dockerCleanup()
         // hack
         dir('worker') {
```

Comment:
@bkabrda are we having a directory worker in the server codebase to clone fabric8-analytics-worker?